### PR TITLE
cartucciam adds hometown of Rome

### PR DESCRIPTION
**Pull request by cartucciam**
This pull request is to add Rome to the list of cities.

- [ ] Does anybody come to Italy for the food?

### CODE REVIEW

**Comment by githubteacher** [comment]
> Does anybody come to Italy for the food?

@cartucciam yes! absolutely! But I want to make sure that I have food that the local like! 🍕

**Comment by awgregory** [comment]
@githubteacher I agree but I think they pizza in Venice was good too!

File: cartucciam-Rome.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Rome, Italy
+
+### Restaurants
+- Da Enzo al 29, Trastevere
```
Comment (cartucciam) [inline cartucciam-Rome.md]: @githubteacher Here you can have real roman food, I'd suggest pasta carbonara and "Coda alla Vaccinara"!

**Comment by cartucciam** [comment]
@awgregory You can find the best pizza in Naples, it's just a couple of hours from Rome.

**Comment by cartucciam** [comment]
@githubteacher @beardofedu what happened here? Could you explain what's the role of @pa101 in this?

**Comment by hiseni** [comment]
@cartucciam pa101 just merged & deleted your PR.
You aren't asked him to, but.. :wink:

**Comment by cartucciam** [comment]
Thanks @hiseni. So there can be more than one merge commit for a single PR?

**Comment by hiseni** [comment]
@cartucciam yep, it's not definitely connected to PR, but to actual branch.
Git stores all the changes in its history, with can even be overwritten by someone.
Checkout best complete (& free!) e-book on git version control here https://git-scm.com/book/en/v2 to level your git skill up :book:

**Pull request merged**